packages/fxa-payments-server/src/components/Tooltip/index.stories.tsx

```diff
@@ -145,53 +143,46 @@ const storyWithMultipleFields = (
     </MockPage>
   );
 
-  if (storyName) story.storyName = storyName;
   return story;
 };
 
 // one field
-export const Default = storyWithOneField(false, 'default');
-export const Dismissible = storyWithOneField(true, 'dismissible');
+export const Default = storyWithOneField(false);
+export const Dismissible = storyWithOneField(true);
 
 // multiple fields
-export const ShowBelow = storyWithMultipleFields(
-  [
-    {
-      showBelow: false,
-      clientHeight: undefined,
-      fieldText: 'showBelow = false',
-    },
-    {
-      showBelow: true,
-      clientHeight: undefined,
-      fieldText: 'showBelow default',
-    },
-    {
-      showBelow: true,
-      clientHeight: undefined,
-      fieldText: 'showBelow = true',
-    },
-  ],
-  'showBelow'
-);
+export const ShowBelow = storyWithMultipleFields([
+  {
+    showBelow: false,
+    clientHeight: undefined,
+    fieldText: 'showBelow = false',
+  },
+  {
+    showBelow: true,
+    clientHeight: undefined,
+    fieldText: 'showBelow default',
+  },
+  {
+    showBelow: true,
+    clientHeight: undefined,
+    fieldText: 'showBelow = true',
+  },
```

Comment:
Q: How is this different from the previous item? Or should L161 have an undefined `showBelow` value?

Reply:
Updated. When `showBelow` is undefined (default) or explicitly set to `true`, the tooltip shows below as expected.